tests-beta/android/MASVS-PLATFORM/MASTG-TEST-0293.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Deep Link Intent Filter Missing android:autoVerify
```

Comment:
This should be "Use of Unverified App Links" plus an additional "Use of Unverified Custom URL Schemes" which is now missing.